books/free-programming-books-langs.md

```diff
@@ -2074,6 +2075,12 @@ That section got so big, we decided to split it into its own file, the [BY SUBJE
 * [The Official Raspberry Pi Project Book 1 (2015)](https://magpi.raspberrypi.com/books/projects-1) (PDF)
 
 
+### RCS
+
+* [RCS — A System for Version Control](https://www.cs.purdue.edu/homes/trinkle/RCShome/rcs.ps) - Walter F. Tichy (PS)
```

Comment:
Fixing file format notes

```suggestion
* [RCS — A System for Version Control](https://www.cs.purdue.edu/homes/trinkle/RCShome/rcs.ps) - Walter F. Tichy (PostScript)
```

Are there any other format easier readable like PDF, ePub...?

Reply:
FreeBSD has a PDF version: https://docs.freebsd.org/44doc/psd/13.rcs/paper.pdf

Reply:
I think that it would be better to replace the .ps link with the GNU link

Reply:
> @davorpa GNU has a PDF version: [gnu.org/software/rcs/tichy-paper.pdf](https://www.gnu.org/software/rcs/tichy-paper.pdf) FreeBSD too: [docs.freebsd.org/44doc/psd/13.rcs/paper.pdf](https://docs.freebsd.org/44doc/psd/13.rcs/paper.pdf)

I agree with Luigi and Eric attending to contrib guidelines, using PDF or ePUB, and even use a gnu.org as target domain. Meanwhile... I'd opt to fix file format notes with the proposed name